monitor.html

```diff
@@ -49,10 +52,44 @@
 	const obs = new OBSWebSocket();
 	
 	// initialize
-	function init() {
+	async function init() {
+		await refreshSettings();
```

Comment:
The ability to have it statically configured without having the python script should still work.
Personally i think the most simple way is to make the script the default behavior, and add a simple true/false variable on the top of the html file where the configuration var's are that simply disables this call.